pkg/cluster/snow.go

```diff
@@ -0,0 +1,81 @@
+package cluster
+
+import anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+
+func snowEntry() *ConfigManagerEntry {
```

Comment:
We should probably add a test parsing one snow cluster
We don't need to reuse the same test as the other providers
It probably make sense to have one specific for snow that only checks snow datacenter and snow machines

Reply:
i will add one in existing parsing test including cluster that covers the Ref to snow objects